site/web/assets/js/search-page.js

```diff
@@ -0,0 +1,216 @@
+const $ = require('jquery');
+const instantsearch = require('instantsearch.js').default;
```

Comment:
Doesn't ES6 import work? `import instantsearch from 'instantsearch.js'` ?

Reply:
Works, works! Have a run and check the console

Reply:
@agathver How would we want to enable webpack to watch our files in production?

Reply:
In production we just run a `yarn build` you need to modify ansible configurations to make sure it is triggered. It is similar to the cache clear task we perform. The `yarn build` must come before generating the Symfony cache.

Reply:
> @aquibbaig Where are you running `yarn dev-server` inside or outside the VM?

`yarn encore dev` inside the vm